environment.yml

```diff
@@ -16,4 +16,14 @@ dependencies:
   - matplotlib
   - alabaster
   - conda-forge::gxx_linux-64
-  - pip
\ No newline at end of file
+  - pip
+  - fastp=0.20.0
+  - fastqc
+  - pathos=0.2.4
+  - pysam=0.15.3
+  - multiqc
+  - python=3
+  - pip:
+    - git+https://github.com/czbiohub/kh-tools.git@olgabot/partition-coding-reads#egg=khtools
+    - git+https://github.com/czbiohub/sourmash.git@olgabot-patch-1#egg=sourmash
```

Comment:
please change it to master, PRs are merged now

Reply:
merged hp encoding changes into sourmash master. lets change this now to 

git+https://github.com/czbiohub/sourmash.git@master#egg=sourmash